Apps/WebClient/src/NewClientApp/src/components/public/waitlist/QueueFullView.vue

```diff
@@ -0,0 +1,32 @@
+<script setup lang="ts">
+import { useRouter } from "vue-router";
+
+import { useWaitlistStore } from "@/stores/waitlist";
+
+const router = useRouter();
+const waitlistStore = useWaitlistStore();
+
+if (!waitlistStore.tooBusy) {
+    router.push({ path: "/" });
+}
+</script>
+
+<template>
+    <v-row justify="center" class="mt-12">
+        <v-col cols="12" md="8" lg="6" xl="4">
+            <v-alert
+                show
+                type="error"
+                data-testid="too-many-tickets"
+                class="mb-12 text-center"
+                text="Can’t access Health Gateway. Please try again later."
```

Comment:
```suggestion
                type="error"
                data-testid="too-many-tickets"
                class="mb-12 text-center"
                variant="outlined"
                border
                text="Can’t access Health Gateway. Please try again later."
```

`show` prop is not needed.  Added common styling for our alerts.